CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 ---
 ## 0.5.1
 
+- Swift Templates are now supported
+
+```
+<% for type in types.classes { %>
+    extension <%= type.name %>: Equatable {}
```

Comment:
Hi @ilyapuchka ,

We need to use something obscure to have the least possible impact on Swift grammar and for it visually stand out from regular swift for ease of human template parsing.

There are two most common families of template grammars. One that use `{{}}`, and one that use `<%%>`.

`{{}}` is more suitable for languages that don't have `{}` as a part of their grammar (HTML).

I've found that `<%` is typically used for template engines that generate C family of languages so I've just used what was proven and common to me.